scss/_card.scss

```diff
@@ -12,6 +12,7 @@ $card-border-radius-inner: ($card-border-radius - $card-border-width);
 $card-cap-bg: #f5f5f5;
 
 .card {
+  background-color: #fff;
```

Comment:
Changed. Is there a reason for the (arguably) odd order of values instead of just using alphabetical which is clear?